src/components/todoInfo.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { Todo } from '../types/Todo';
+import { ModalOverlay } from './modalOverlay';
+
+type Props = {
+  todoInfo: Todo,
+  addComplitedTodo: (id:number)=> void
+};
+
+export const TodoInfo: React.FC<Props> = ({ todoInfo, addComplitedTodo }) => {
+  const [isTodoEditing, setIsTodoEditing] = useState(false);
+  const [editedValue, setEditedValue] = useState('');
+
+  const {
+    id,
+    title,
+    completed,
+  } = todoInfo;
+
+  const onInputChange = ({ target }:React.ChangeEvent<HTMLInputElement>) => {
+    setEditedValue(target.value);
+  };
+
+  const onInputBlur = () => {
+    setIsTodoEditing(false);
```

Comment:
You don't need to implement editing in this part of the task. And btw it doesn't work now.

Reply:
I need to so set this  setIsTodoEditing, It works in this way, I dont takes edited values or etc, so there are no functional for this